.github/workflows/e2e-all-k8s.yml

```diff
@@ -0,0 +1,40 @@
+---
+name: End to End All K8s
+
+on:
```

Comment:
I think this should be general job and not a PR job which runs weekly/twice a week on the devel branch.

Reply:
Do you mean this should run on every PR? There is other CI for that, see: https://github.com/submariner-io/submariner-operator/blob/devel/.github/workflows/e2e-full.yml#L33

This is one of the jobs discussed as a part of https://github.com/submariner-io/submariner-website/issues/624, for the use-case of wanting to run the full E2E matrix against every K8s version when developing something that is expected to have an impact, periodically, and on releases.

Reply:
I meant to just keep the `schedule` section and remove `pull_request` section but then saw your PR in `release` repo https://github.com/submariner-io/releases/pull/298 and understood the intent.